1.12.2-/src/main/resources/fabric.mod.json

```diff
@@ -1,7 +1,7 @@
 {
 	"schemaVersion": 1,
 	"id": "discotemplate",
-	"version": "1.0.0",
+	"version": "${mod_version}",
 	"name": "Test",
 	"description": "A disco template!",
 	"authors": [
```

Comment:
if we're already making a proper template, then let's also extern all of these properties here. Copy the processResources, gradle.properties and fabric.mod.json from LoTAS or smth